src/game/Bomberman.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "Entity.hpp"
+
+#include <memory>
+
+#include "physics/PhysicsId.hpp"
+#include "physics/PhysicsEngine.hpp"
+#include "graphics/RendererId.hpp"
+#include "graphics/RendererPool.hpp"
+#include "HumanPlayer.hpp"
+
+class Bomberman : public Entity
+{
+public:
+    void update(DeltaTime) override;
+    bool areYouDead() const override;
+
+private:
+    physics::PhysicsId physicsId;
```

Comment:
consider rearranging membres in way 
- normal members
- unique_ptrs
- shared_ptrs

Reply:
Test3